.travis.yml

```diff
@@ -52,12 +52,16 @@ addons:
       - wget
 python:
   - 2.7
-rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.0
-  - jruby-1.7.27
-#  - jruby-head
+matrix:
```

Comment:
@pepijnve One major difference when using `matrix` is that `gemfile` is now only applied to the first stage (ie. `rvm: 1.9.3`).
In other words, the library will only be tested using both `Gemfile` and `Gemfile.master` with Ruby 1.9.3. For other Ruby versions, we will only use `Gemfile`.

If you think that's not acceptable, then I can try to tweak the Travis build file to build the library using  both `Gemfile` and `Gemfile.master` for every Ruby version.

Reply:
I forgot how we puzzled this together. The intention is to build with whatever Ruby versions make sense and to be able to build against the latest released asciidoctor and the current master.
If asciidoctor master is only used with one particular Ruby version then we should probably reorder things so that that build uses Ruby 2.something. If asciidoctor dropped 1.9 support on master it doesn't make sense to test master with that.

Reply:
Ok :+1: 

I don't want to introduce too much changes at once, so I will create a follow-up pull request where I run Travis against recent versions of Ruby (against both Asciidoctor 1.5.8 and Asciidoctor master).